modules/programs/neomutt.nix

```
@@ -223,9 +223,11 @@ let
 
       # Extra configuration
       ${account.neomutt.extraConfig}
-    '' + optionalString (account.signature.showSignature != "none") ''
+    '' + (if (account.signature.showSignature == "none") then ''
```

If you prefer using this code then can remove the unnecessary parentheses:
```suggestion
    '' + (if account.signature.showSignature == "none" then ''
```